Adopted Strategic Goals, wording tweaks.  Fixes #19.

cwilso opened the pull request:
Avneesh and Tzviya have agreed the Strategic Goals should be managed here, so I pulled them in.  We can now work on wordsmithing them here, through issues and PRs.

dwsinger commented on README.md:
```diff
@@ -23,19 +23,36 @@ The W3C itself must function with these strategic principles:
 * Put the needs of users first: above authors, publishers, implementers, paying W3C Members, or theoretical purity
 * Ensure that the Web can work for everyone, striving for diversity & inclusion of participants from different geographical locations, cultures, languages, accessibility needs, gender identities, and more. Continue to champion accessibility and internationalization as core principles for the Web.
-* Make the web trustworthy, by improving security and privacy for users. We must revisit the current Web platform and explore how to responsibly improve personal privacy for users.
+* Make the web trustworthy, by improving security and privacy for users. We must revisit the current Web platform and explore how to responsibly improve personal privacy for users. 
```
we could avoid the split infinite and make it more active by saying "how we can responsibly improve personal privacy"

dwsinger commented on README.md:
```diff
@@ -23,19 +23,36 @@ The W3C itself must function with these strategic principles:
 
 # Our Purpose: Focusing on the Integrity of the Web
 
+We believe the W3C's fundamental purpose is to:
```
This entire document is a statement of belief; 'fundamental' is a word I'd only use if I needed to contrast with something (e.g. ancillary).

"The W3C must/will:"

dwsinger commented on README.md:
```diff
@@ -23,19 +23,36 @@ The W3C itself must function with these strategic principles:
+    * We are socially responsible and committed to ensure that web is for everyone, we greatly emphasize accessibility, internationalization, privacy, security and diversity.
+    * We are committed to developing open and royalty free standards with high focus on interoperability and collective empowerment.
+
+# Appendices & References
```
Acknowledgements and Supporting Material

dwsinger reviewed (approved):
LGTM modulo some minor suggested changes in-line

michaelchampion reviewed (commented):
I'm OK with merging these changes but we need to iterate more on the language and structure.  Some general comments:

First, the heading says "goals" (which implies WHAT statements) but the bullet points  talk about HOW to pursue  the goals.  I'd suggest rephrasing, e.g. "everyone can use the web regardless of physical ability or language and culture", "users are confident in the web's integrity with confidence in its security and respect for privacy", "the interests and perspectives of currently underrepresented stake holders such as end users, content creators, website developers, etc. are fairly considered in evolving the web platform", etc.  

- There are 9 bullets, that's too many for the target audience to process and remember without close, repeated readings.  Reduce them to the 3 to 5 most important points you want the reader to take away.  

One way to do this would be byremoving those that are directed at an internal rather than external audience ; "Establish the unique global brand identity for W3C" is just not something that someone without strong affinity for the current organization is likely to care about. 

Another way would be by combining similar points, such as "Further improve the environment..." and "establish collaborative relationship ...".  Yes, they are saying different things, but the larger goal is the same: W3C is a community where diverse member organizations, other standards and open source bodies, and user-focused people and groups can work together fairly and effectively to improve web standards.  

Also "Evolve the extensible web architecture to empower industries and individuals to address the evolving user’s needs" and "Reinforce efforts on incubation, make it more structured and improve consensus building among the key stake holders" are more or less the same goal: Evolve the platform to address changing user needs in an inclusive, consensus-driven manner.  

Here's some text implementing these suggestions, more to illustrate the points than as something ready to offer as a PR:

## Strategic Goals

 * everyone can use the web regardless of physical ability or language and culture, 
 * users are confident in integrity of the web platform, especially its ability to protect their security and respect their privacy
 *  currently underrepresented stakeholders such as end users, content creators, website developers, etc. feel that their interests are considered in evolving the web platform
 * The web platform evolves to meet changing stakeholder needs in an inclusive, consensus-driven manner.
 * W3C is recognized as a community where diverse member organizations, other standards and open source bodies, and user-focused people and groups can work together fairly and effectively to improve web standards.

cwilso commented:
@michaelchampion I moved your comments to a different issue so we don't lose them.

tantek reviewed (commented):
"Ensure the Web does not favor centralization. " is a negative goal amongst a list of positives. It needs rephrasing/reframing as a positive, as something actionable. Unfortunately saying "favor decentralization" doesn't actually help because 1) "decentralization" itself is negative framing, and 2) that term has been colonized by blockchainers.

tantek reviewed (changes requested):
This is a good first cut at incorporating "Strategic Goals". There are a bunch of minor grammar fixes needed (which I'm happy to copy-edit after this is merged), and there are a few content aspects which could be improved. 
"Reinforce" doesn't really fit "incubation" because it's far from a well understood thing that just needs "reinforcement".  
"Establish collaborative relationship" is also inaccurate as there are many *existing* relationships with other organizations in the "domain of internet and web standards", and "establish" makes it sound like this is ignoring/neglecting/erasing those existing relationships which is a bad look.
The whole "brand identity" item — is that really a *Strategic* Goal? I find myself sympathetic (nodding) at the subpoints, except the first two are duplicates of previous Strategic Goals points, and the third could be written to stand on its own.

cwilso commented:
OK, @tantek I think I've addressed your issues.  I pulled the Brand identity out as a separate section for now.  We may want to elide it or move to another doc, but I didn't want to delete it yet.

tantek reviewed (changes requested):
While I strongly agree with the spirit of "Ensure the Web does not favor centralization.", it should be dropped from this pull request as it's distracting from / unrelated to the main goal of incorporating the Strategic Goals, and is problematic as previously commented above.

Separating "Identity" to its own section helps. It's still a bit awkward/tentatively worded, and feels like it is trying to be both strong in current aspects and simultaneously aspirational which I'm not sure makes sense for an "Identity" section (the "socially responsible" and "diversity" additions are very good in spirit, but feel "bolted on" to the a11y, i18n, priv, sec quartet. Similarly to the above, I'd prefer if this "Identity" section was moved to its own pull request (for further iteration) as it's not essential to getting the Strategic Goals merged in.

avneeshsingh commented:
Would like to remind to add some principles that we discussed in our last call:
- Balance: Standards activities are not exclusively dominated by any particular person, company or interest group.
- Transparency, equity and fairness.
- 
After reading everything together, couple of more comments for principles.
- Incubation fits better in goals, instead of principles. Or we can combine it with some other principle.
- When we say that web should not be centralized. It would be better to phrase it differently to avoid "not", negative statement.

I also went through patch at https://github.com/WebStandardsFuture/Vision/tree/patch
- I think goal 3 and 4 can be combined. But goal 1 and 2 should be separate so that privacy - security and a11y + i18n are well emphasized.
- It is good to have separate heading for building brand for new W3C.
